docs/docs/sql/structure.md

```diff
@@ -1,11 +0,0 @@
-# High-level program structure
```

Comment:
Why was this removed?

Reply:
This information is duplicated in the guide.